### PR TITLE
fix(components): [pagination] fix type error

### PR DESCRIPTION
**Pull request by ryuhangyeong**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by btea — APPROVED** [review]

File: packages/components/pagination/src/components/sizes.ts [hunk 1]
```diff
@@ -16,7 +22,8 @@ export const paginationSizesProps = buildProps({
   },
   disabled: Boolean,
   size: {
-    type: String,
+    type: String as PropType<ComponentSize>,
+    validator: isValidComponentSize,
```
Comment (holazz) [inline packages/components/pagination/src/components/sizes.ts:26]: **Suggestion**

```ts
import { componentSizes } from '@element-plus/constants'
```

```ts
size: {
  type: String,
  values: componentSizes,
},
```

Comment (ryuhangyeong) [reply packages/components/pagination/src/components/sizes.ts]: thank you for your review!

**Comment by holazz** [comment]
`default: 'default'` also seems to be removed.

**Comment by ryuhangyeong** [comment]
@holazz done!

**Review by holazz — APPROVED** [review]

**Pull request merged**